admin/make-release.zsh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env zsh
+
+set -eux
+
+tag=$1
+
+mkdir -p releases
```

Comment:
It could be a little bit safer to make sure that the working directory is under control and does not rely on a way the script is launched (e.g. as `./make-release` inside `admin/`). For instance, we could do the following chdir before the operations: `cd ${0%/*}/..`.